README.md

```diff
@@ -1,4 +1,7 @@
 # Docker Mailserver
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
Can this be disabled?

Reply:
I too think that this should not be there. Your suggestion from above seems to to into the right direction.